static/downloadables/grep_file.py

```diff
@@ -157,51 +157,56 @@ def filterTXT(tableObjDict, clumpsObjDict, inputFilePath, filteredFilePath, trai
 
 
 def filterVCF(tableObjDict, clumpsObjDict, inputFilePath, filteredFilePath, traits, studyIDs, studyTypes, ethnicities, isAllFiltersNone, p_cutOff):
-    # open the input vcf
-    vcf_reader = openFileForParsing(inputFilePath, False)
-    # open the filtered input path for writing out the lines from the input vcf that match the given filters
-    vcf_writer = vcf.Writer(open(filteredFilePath, 'w'), vcf_reader)
+    # open the input file path for opening
+    inputVCF, compressed = openFileForParsing(inputFilePath, False)
```

Comment:
Touch bases with Matt. He has a pull request open for some work on zip files and I'm worried that some of the changes you have here won't work with his changes. (@MattCloward)